jenkins/bootstrap_test.py

```diff
@@ -1892,6 +1892,11 @@ def testValidJobEnvs(self):
                         self.assertTrue(
                             os.path.isfile(path),
                             '%s does not exist for %s' % (path, job))
+                    elif 'kops' not in job:
+                        m = re.match(r'--cluster=([^\"]+)', arg)
+                        if m:
+                            cluster = m.group(1)
+                            self.assertLessEqual(len(cluster), 20, 'Job %r, --cluster should be less than 20 chars' % job)
```

Comment:
assertLess(20), failed, what? oops, longest is exactly 20! assertLessEqual then, without change the message :-)